src/asciidoc/docs/distribute/app-store.adoc

```diff
@@ -0,0 +1,86 @@
+= Listing in the App Store
+:page-slug: app-store
+:page-description: An introduction to the Koji App Store and how to list your Koji app.
+:includespath: ../_includes
+
+After you create, test, and move your app to production, you can share its production URL or list the app in the Koji App Store.
```

Comment:
“Move to production” - I’m not sure what this phrase means here. I know we haven’t decided on a replacement term for “publish” yet (or changed how the platform actually works). So, maybe it’s best to use "publish" for now. At least it might be easier to find and replace when we do make the change :)

Reply:
Fixed.